addons/binding/org.eclipse.smarthome.binding.yahooweather/ESH-INF/thing/thing-types.xml

```diff
@@ -27,6 +27,18 @@
                 <description>Specifies the refresh interval in seconds.</description>
                 <default>60</default>
             </parameter>
+
+            <action name="location" type="text">
```

Comment:
Now, I got it!
This says: action should be named like the the parameter/-group it's going to provide edit support for.

Have a look at the redundancy this solution would ends in: 
```
        <config-description>
            <parameter name="location" type="text" required="true">
                <label>Location</label>
                <description>Location for the weather information.
                    Syntax is WOEID, see https://en.wikipedia.org/wiki/WOEID.
                </description>
            </parameter>
            <parameter name="refresh" type="integer">
                <label>Refresh interval</label>
                <description>Specifies the refresh interval in seconds.</description>
                <default>60</default>
            </parameter>

            <action name="location" type="text">
                <label>Location</label>
                <description>Location for the weather information.
                    Syntax is WOEID, see https://en.wikipedia.org/wiki/WOEID.
                </description>
            </action>
            <action name="refresh" type="integer">
                <label>Refresh interval</label>
                <description>Specifies the refresh interval in seconds.</description>
                <default>60</default>
            </action>
        </config-description>
    </thing-type>
```

All the labels, descriptions, and other attributes are duplicated. 
Can't it be expressed like this?
```
<config-description>
    <parameter name="location" type="text" required="true">
        <label>Location</label>
        <description>...</description>
    </parameter>

    <action name="updateLocation">
        <label>Update Location</label>
        <parameters>
           <parameter>location</parameter>
        </parameters >
    </action>
</config-description>
```